components/supervisor/pkg/ports/ports_test.go

```diff
@@ -23,6 +23,8 @@ import (
 )
 
 func TestPortsUpdateState(t *testing.T) {
+	//	t.Skip("for now")
```

Comment:
we need to enable and fix them before merging?

Reply:
fixed